standards/source-control.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title:  "Source Control"
+---
+
+# Source Control
+
+At the Apprenticeship Service all our source code is [open by default](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable), and stored on well-known, public code hosting services [in line with GDS Service Manual guidance](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable).
+
+## GitHub
+
+Our current source-control platform is [GitHub](https://github.com/SkillsFundingAgency/) where we are part of the SkillsFundingAgency organisation. All new repositories for products and services must be created in the [Skills Funding Agency organisation](https://github.com/SkillsFundingAgency), this includes all service production code and prototypes. 
+
+> Any work created outside of the Skills Funding Agency organisation should be transferred into the Skills Funding Agency organisation at the earliest opportunity. [Guide to transferring a repository](https://help.github.com/en/articles/transferring-a-repository).
+
+It is accepted practice to access the SkillsFundingAgency GitHub organisation via personal GitHub accounts but before you can contribute to repositories in this organisation it is necessary to be granted the appropriate security role.
+
+## Secrets and Sensitive Configuration
+
+To ensure the safety and security of our systems; secrets, sensitive configuration and artefacts should not be pushed to public repositories. However this does not mean that private repositories are a good way to protect secrets, they are not. Secrets should be stored using technologies specifically designed for the task such as Azure Key Vaults, Secure Azure DevOps pipeline secret variables, etc...
+
```

Comment:
During development, settings files that are ignored by git - i.e. `appsettings.Development.json` - may be a convenient way to store secrets without them being accidentally added to the repository

Reply:
Happy to add this in